astroid/brain/brain_namedtuple_enum.py

```diff
@@ -195,25 +193,18 @@ def infer_named_tuple(
     node: nodes.Call, context: InferenceContext | None = None
 ) -> Iterator[nodes.ClassDef]:
     """Specific inference function for namedtuple Call node."""
-    tuple_base_name: list[nodes.NodeNG] = [
-        nodes.Name(
-            name="tuple",
-            parent=node.root(),
-            lineno=0,
-            col_offset=0,
-            end_lineno=None,
-            end_col_offset=None,
-        )
-    ]
+    tuple_base = util.safe_infer(_extract_single_node("tuple"))
```

Comment:
Why are you inferring here? Shouldn't this just be the `nodes.Name` for `tuple`? Similar to the comment you make about `EnumMeta` preferably being the `Name` reference?

Reply:
My motivation for using the actual definition instead of a Name was to avoid the situation where the actual tuple is shadowed, and thus the Name refers to something different. 

However, looking at it my current implementation doesn't achieve that. A better way to do it would be something like `tuple_base = builtin_module["tuple"]`. If you think that my worry above is weakly-founded, I can revert to a `Name`

Reply:
(I talk about it in some more detail in the commit message)

Reply:
Hmm, actually it appears that `tuple_base: nodes.Name = _extract_single_node("tuple")` works pretty well. It's not going to be shadowed, because it belongs to an empty module. Should we go with that?